snap/gui/org.freecadweb.FreeCAD.desktop

```diff
@@ -0,0 +1,19 @@
+[Desktop Entry]
+Name=FreeCAD (realthunder)
+Name[de]=FreeCAD (realthunder)
+Name[pl]=FreeCAD (realthunder)
+Name[ru]=FreeCAD (realthunder)
```

Comment:
I added a modified desktop file with your nick included in the app name. This way, one can distinguish it from other FreeCAD installs on the same computer. Let me know if that makes sense to you.

Reply:
Maybe something like [this](https://github.com/realthunder/FreeCADMakeImage/blob/master/conda/branding/asm3/AppDir/freecad_link.desktop), and [this](https://github.com/realthunder/FreeCADMakeImage/blob/master/conda/branding/asm3-daily/AppDir/freecad_link.desktop) for the edge?

Reply:
I'll just pull in your branding from https://github.com/realthunder/FreeCADMakeImage/tree/master/conda/branding, including the desktop file